pom.xml

```diff
@@ -49,7 +49,7 @@
     <parent>
         <groupId>org.jahia.modules</groupId>
         <artifactId>jahia-modules</artifactId>
-        <version>7.2.1.0</version>
+        <version>7.5.0.0-SNAPSHOT</version>
```

Comment:
You should then increment the major version number of this module. This is a breaking change since this module will not require "Jahia 8". Please note this was planned as part of which was planned as part of https://jira.jahia.org/browse/BACKLOG-13160.

I am not quite sure to understand why you add to update the parent pom as part of this ticket though.

Reply:
I thought it was necessary, because those icons are part of V8 only
if it will be done in the other story then ok, I'll remove the version and let it for the other story

Reply:
done